src/System.Private.CoreLib/System.Private.CoreLib.csproj

```diff
@@ -48,13 +45,6 @@
       <Visible>false</Visible>
     </AssemblyMetadata>
   </ItemGroup>
-  <!-- Platform specific properties -->
-  <PropertyGroup Condition="'$(Platform)' == 'amd64'">
-    <PlatformTarget>x64</PlatformTarget>
-    <Prefer32Bit>false</Prefer32Bit>
-    <BaseAddress>0x180000000</BaseAddress>
-    <DefineConstants>BIT64;AMD64;$(DefineConstants)</DefineConstants>
```

Comment:
So where do these defines get defined for AMD64 now that you have deleted this block?